README.rst

```diff
@@ -30,7 +30,7 @@ Install the package with pip:
 
 ::
 
-    pip install -e cf-python-logging-support git+https://github.com/SAP/cf-python-logging-support.git
+    pip install -e sap_cf_logging git+https://github.com/SAP/cf-python-logging-support.git
```

Comment:
Does this work, my pip install says -e needs to be followed by the url

Reply:
the idea is after this change is merged we change the command to use pypi

Reply:
pip install -e git+git://github.com/SAP/cf-python-logging-support.git#egg=sap_cf_logging